targets/maixbit.ld

```diff
@@ -0,0 +1,11 @@
+
+MEMORY
+{
+    RAM (xrw) : ORIGIN = 0x80000000, LENGTH = (6 * 1024 * 1024)
```

Comment:
I didn't know such an expression was allowed, but you can make the code more readable with `6M` (meaning 6 megabytes).

Reply:
done